File: Cpp/Processor.cpp
class Processor {

    /* Private Attribute */
    private:

        string name;
        int price;

    /* Public Attribute and Method */
    public:

        /* Constructor */
        Processor() {
            this->name = "Name";
            this->price = 0;
        }

        Processor(string name, int price) {
            this->name = name;
            this->price = price;
        }

        /* Setter dan Getter Name */
        void setName(string name) {
            this->name = name;
        }

        string getName() {
            return this->name;
        }

        /* Setter dan Getter Price */
        void setPrice(int price) {
            this->price = price;
        }

        int getPrice() {
            return this->price;
        }

        /* Destructor */
        ~Processor() {

        }

};

File: php/Processor.php
<?php

class Processor {
    
    /* Private Attribute */
    private $name = "Name";
    private $price = 0;

    /* Constructor */
    public function __construct($name = "Name", $price = 0) {
        $this->name = $name;
        $this->price = $price;
    }

    /* Setter dan Getter Name */
    public function setName($name) {
        $this->name = $name;
    }

    public function getName() {
        return $this->name;
    }

    /* Setter dan Getter Price */
    public function setPrice($price) {
        $this->price = $price;
    }

    public function getPrice() {
        return $this->price;
    }

    /* Destructor */
    public function __destruct() {

	}

}

?>